docs/scenarios_authoring/HandlingSchemas.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 11
+---`
+
+# Handling Schemas
+
+## Overview
+
+Nussknacker supports two types of schema:
+* [JSON Schema](https://json-schema.org/) in version: `Draft 7`
+* [Avro Schema](https://avro.apache.org/) in version: `1.11.0`
+
+Avro Schema and JSON Schema are available on [Streaming Source/Sink](/docs/scenarios_authoring/DataSourcesAndSinks.md)
+in contrast to [RequestResponse Source/Sink](/docs/scenarios_authoring/RRDataSourcesAndSinks.md) where available is just only one - JSON Schema.
+
+Nussknacker converts `source schema` to its own static `Typing` information to create hinting on the FE Designer,
+whereas `sink schema` is used to validate output data.
+
+## Source conversion
+Conversion at source to specific type means that behind the scene Nussknacker converts `primitive type` to `logical type`
+consequently, the end-user has access to object's methods, e.g. let's imagine the schema contains the field `foo` with logical type `Date`.
+Originally event contains data with the field `foo` where is stored `Long`. After conversion, field `foo` will be converted to `Date`,
+and the end user on Designer will have access to `#input.foo` with type `Date` and can do `#input.foo.toInstant`.
+
+
+## Avro Schema
+
+### Source supports & conversions
```

Comment:
This looks like very important, although being a scenario author, I am not sure what to do with this information - I would add a sentence of explanation. 

The topic of data types is covered [here](https://nussknacker.io/documentation/docs/scenarios_authoring/Spel#basic-primitive-data-types) - I wonder whether it would be better to put the stuff below to SpEL Cheat Sheet. Ask mproch or arkadius for an opinion